pkg/helper/options/delete.go

```diff
@@ -28,24 +28,31 @@ type DeleteOptions struct {
 	ParentResource     string
 	ParentResourceName string
 	ForceDelete        bool
-	DeleteAll          bool
+	DeleteRelated      bool
+	DeleteAllNs        bool
```

Comment:
`DeleteAllNs` since there will be ClusterTask case where it's not focused on deleting all resources in a namespace.